src/backend/expungeservice/expunger.py

```diff
@@ -83,15 +83,10 @@ def run(record: Record, today: date = date.today()) -> Dict[str, TimeEligibility
                 )
 
             if charge.convicted() and most_recent_blocking_conviction:
-                conviction_string = "other conviction" if charge.convicted() else "conviction"
-                summary = most_recent_blocking_conviction.case(cases).summary
-                potential = "potential " if not summary.closed() else ""
-                eligibility_dates.append(
-                    (
-                        most_recent_blocking_conviction.disposition.date + relativedelta(years=10),
-                        f"137.225(7)(b) – Ten years from most recent {potential}{conviction_string} from case [{summary.case_number}].",
-                    )
-                )
+                relative_case_summary = most_recent_blocking_conviction.case(cases).summary
+                blocking_convictions_time_eligibility = Expunger._other_blocking_conviction_years_by_level(charge.level, most_recent_blocking_conviction.disposition.date, relative_case_summary)
+                if blocking_convictions_time_eligibility:
```

Comment:
When will `blocking_convictions_time_eligibility` resolve to `False`/`None` here?

Reply:
Ah, it's always non-empty because of the check above on `most_recent_blocking_conviction`, so I'll remove the conditional here.